language/Portuguese_Brazil/strings.po

```diff
@@ -1727,7 +1727,7 @@ msgstr "MAMEinfo"
 
 #: src/frontend/mame/ui/selmenu.cpp:132
 msgid "MARPScore"
-msgstr ""
+msgstr "MARPScore"
```

Comment:
It's the name of a web site - it can load scores scraped from this site: http://replay.marpirc.net/